call-of-papers/src/pages/Lectures/LecturesList.js

```diff
@@ -1,5 +1,94 @@
 import React from 'react';
+import { Layout, List} from 'antd';
+import './lectures-list.css';
+import data from './lectures-list-test.json';
 
-const LecturesList = () => <h2>Lectures List</h2>;
+
+// const data = [
```

Comment:
Porque está comentado?
Se não for usar, pode apagar depois =)

Reply:
Eu decidi jogar em outro arquivo, podia jurar que tinha apagado!